_events/2022/2022-10-funder-talk-series.md

```diff
@@ -18,18 +18,14 @@ US-RSE is happy to be starting a new series of conversations with funders for 1)
 
 ## Event
 
-The inaugural talk of the US-RSE Funder Talk Series will be with Hal Finkel, Ph.D. from the US Department of Energy's (DOE's) Office of Advanced Scientific Computing Research (ASCR). This event will take place **Monday, October 17th at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**
-
-
-Please join us in this interactive conversation. (If you can't participate in the interactive event, where you will be able to ask questions and help funders understand US-RSE, it will also be recorded.)
-
-[**Register on Zoom**](https://boisestate.zoom.us/meeting/register/tJckd-2vqjIoGNRrjGPruBRxQBthv5Maidmm)
+The inaugural talk of the US-RSE Funder Talk Series will be with Hal Finkel, Ph.D. from the US Department of Energy's (DOE's) Office of Advanced Scientific Computing Research (ASCR). This event took place **Monday, October 17th at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**
```

Comment:
```suggestion
The inaugural talk of the US-RSE Funder Talk Series featured Hal Finkel, Ph.D. from the US Department of Energy's (DOE's) Office of Advanced Scientific Computing Research (ASCR). This event took place **Monday, October 17th at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**
```

Reply:
Just one language thing - otherwise, fabulous!

Reply:
Committed. No idea why this is still listed as 'change suggested' below. I checked before the last request and it was changed to featured.